src/components/organisms/Dashboard.tsx

```diff
@@ -47,19 +47,15 @@ const Dashboard: React.FC = () => {
                                 <ParentSize>
                                     {(parent) => (
                                         <DataWrapper query={item.query}>
-                                            {({ data, loading, error }) => (
-                                                <>
-                                                    {loading && <Text>Loading...</Text>}
-                                                    {error && <Text>{error.message}</Text>}
-                                                    {data && (
-                                                        <ThresholdChart
-                                                            data={data}
-                                                            height={parent.height}
-                                                            width={parent.width}
-                                                        />
-                                                    )}
-                                                </>
-                                            )}
+                                            {({ data }) =>
+                                                data.length !== 0 && (
```

Comment:
Is this a necessary check?

In the case where it's zero you are we are now rendering _nothing_ instead of an empty chart.

Reply:
You could also perform a check in the `DataWrapper` component to see if data is a falsey value.

Reply:
ThresholdChart does not work with an empty array, so it will throw an error

Reply:
OK. Perhaps it would be wise to show an error if the data array is empty?

If you don't have capacity to fix this in this PR, you can create an issue for it.

Reply:
new issues #50